### PR TITLE
Capture guidance

### PR DESCRIPTION
**Pull request by stevenwilkin**
First cut of capturing the details of the delivered guidance. Just capturing the attendee's name currently.

### CODE REVIEW

**Comment by andrewgarner** [comment]
How about `GuidanceRecord` instead of `Guidance`?

**Comment by andrewgarner** [comment]
Or maybe `AppointmentRecord`?

**Comment by stevenwilkin** [comment]
Any opinions @gjvis ?

File: features/support/pages/guidance_page.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class GuidancePage < SitePrism::Page
+  set_url '/guidance/new'
+
+  element :name, 'input.t-name'
+  element :submit, 'input[type="submit"]'
```
Comment (andrewgarner) [inline features/support/pages/guidance_page.rb]: We mixing our approach here. We're using test CSS classes and CSS attribute selectors. Why not `input.t-submit`?

File: features/step_definitions/record_of_guidance_steps.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+When(/^appointment details are captured$/) do
+  @page = GuidancePage.new
```
Comment (andrewgarner) [inline features/step_definitions/record_of_guidance_steps.rb]: We don't need an instance of the page, we're only using it within this method.

Comment (stevenwilkin) [inline features/step_definitions/record_of_guidance_steps.rb]: Good point. My first thought was that we'd need access to it in the other step definitions but on looking at them again, the rest of them don't involve the guidance capture form

File: features/support/pages/guidance_page.rb (hunk 1 above)
Comment (stevenwilkin) [inline features/support/pages/guidance_page.rb]: Sure, I've no argument not to use CSS classes to target all the elements on the page

**Comment by stevenwilkin** [comment]
Model renamed to ``AppointmentSummary`` per discussion

**Comment by stevenwilkin** [comment]
RuboCop!

**Pull request merged**